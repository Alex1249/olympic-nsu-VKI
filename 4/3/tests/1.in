5
8824
64576
2554
16024
10138624
