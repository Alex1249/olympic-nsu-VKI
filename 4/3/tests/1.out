8
64
2
8
1024
